### PR TITLE
Add a doc about contributing and issue and pull request templates.

### PR DESCRIPTION
**Pull request by JuliaJakubcova**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# How to contribute
+
+In the first place, thank you for thinking about contributing to KMMBridge!
+Here you can find a set of guidelines for pitching in.
+
+## Questions
+
+If you have any questions, please, contact us in the Kotlin [Community Slack](https://kotlinlang.slack.com/) in <channel-name> channel. To join the Kotlin Community Slack, [request access here](http://slack.kotlinlang.org/).
```
Comment (JuliaJakubcova) [inline CONTRIBUTING.md:8]: @samhill303 Can you provide a link for slack channel where KMMBridge questions can be asked, please?

Comment (samhill303) [reply CONTRIBUTING.md:8]: #touchlab-tools https://kotlinlang.slack.com/archives/CTJB58X7X

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+## Summary
+<!--- Copy summary from issue link or write a shortened description of it -->
+
+## Fix
+<!-- What did you do to fix the issue? -->
+
+## Testing
+<!-- Remove any lines that were not performed -->
+- `./gradlew test`
+- `./gradlew :kmmbridge:build`
```
Comment (russhwolf) [inline .github/PULL_REQUEST_TEMPLATE.md:15]: This can be reduced to just `./gradlew build`.

**Review by russhwolf — APPROVED** [review]

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# How to contribute
+
+In the first place, thank you for thinking about contributing to KMMBridge!
+Here you can find a set of guidelines for pitching in.
+
+## Questions
+
+If you have any questions, please, contact us in the Kotlin [Community Slack](https://kotlinlang.slack.com/) in [this channel](https://kotlinlang.slack.com/archives/CTJB58X7X). To join the Kotlin Community Slack, [request access here](http://slack.kotlinlang.org/).
```
Comment (russhwolf) [inline CONTRIBUTING.md:8]: Maybe "... in the [touchlab-support](...) channel ..." so it's recognizable without needing to click through the link

**Pull request merged**